Source/WebKit/Shared/DocumentEditingContext.mm

```diff
@@ -58,24 +82,38 @@ static inline NSRange toNSRange(DocumentEditingContext::Range range)
     }
 
     [platformContext setSelectedRangeInMarkedText:toNSRange(selectedRangeInMarkedText)];
+    setOptionalEditingContextProperties<UIWKDocumentContext>(*this, platformContext.get(), options);
```

Comment:
Is it possible to skip the template arg here because CTAD from `platformContext.get()`?

Reply:
Good point! Let me give that a try..